src/hooks/useInMemoryPaging.js

```diff
@@ -29,12 +28,6 @@ const useInMemoryPaging = ({
 	const currentPage = viewState.currentPage ?? 1;
 	const nextPageToLoad = viewState.nextPageToLoad ?? 1;
 
-	if (internalSortAndFilterFn.current !== sortAndFilterFn) {
```

Comment:
It was a nice "warning" mechanism, any reason why you removed it?

Reply:
I have a future use case in mind where the sortAndFilterFn prop may change between render. It will not change for every render but it will be more than once.